pantable/pantable.py

```diff
@@ -177,14 +182,13 @@ def read_data(include, data):
     Return None when the include path is invalid.
     """
     if include is None:
-        if py2:
-            data = data.encode('utf-8')
-        io_universal = io.BytesIO if py2 else io.StringIO
-        with io_universal(data) as file:
+        with io.StringIO(data) as file:
             raw_table_list = list(csv.reader(file))
     else:
         try:
-            with open(str(include)) as file:
+            # Handle bad yaml meta-data input by trying to make a string out of
+            # include, so we can open it as a file or get an IOError.
+            with io.open(str(include)) as file:
```

Comment:
On python3, open is the same as io.open.

You might need this on Python2, but `pasteurize` probably done this automatically.